packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -581,6 +596,10 @@ Then('I click the submit button', () => {
   }).click();
 });
 
+Then('I click the label containing text {string}', (labelText: string) => {
```

Comment:
Ideally we would click on an input/button but since the `Overwrite existing files` checkbox compossible contains label+span+svg, we're using this approach